content/tutorials/configuration.md

```diff
@@ -320,4 +360,8 @@ keystore:
     path: ./keys # The key store directory. Keys will be created inside ./keys/
 ```
 
-For production setups, only secure key stores backed by a KMS such as [Hashicorp Vault](https://github.com/minio/kes/wiki/Hashicorp-Vault-Keystore). 
\ No newline at end of file
+For production setups, only secure key stores backed by a KMS such as [Hashicorp Vault](https://github.com/minio/kes/wiki/Hashicorp-Vault-Keystore). 
```

Comment:
Maybe this was intended to be `only use secure key stores` ?

Reply:
Cleaned up